daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -550,6 +550,8 @@ createDamlAppTests = testGroup "create-daml-app" [gettingStartedGuideTest | not
         withCurrentDirectory tmpDir $ do
           callCommandSilent "daml new create-daml-app create-daml-app"
         let cdaDir = tmpDir </> "create-daml-app"
+
+        -- First test the base application (without the user-added feature).
```

Comment:
This test is quite long now. I don’t think it makes much sense to split it up but I think it would be nice to use `testCaseSteps` to make the individual steps a bit more explicit.

Reply:
Done now